stdlib/asm/crypto/fri/helper.masm

```diff
@@ -170,17 +171,21 @@ export.load_and_verify_remainder
     push.0.0.0.0
     adv_loadw
     exec.constants::tmp7 mem_storew
+    #=> [COM, ...]
 
     # Reseed with remainder commitment
     exec.random_coin::reseed
+    #=> [...]
 
     # adv_pipe the remainder codeword
     ## Get the length of remainder
-    exec.constants::tmp6 mem_loadw
+    padw exec.constants::tmp6 mem_loadw
```

Comment:
Was this a bug previously?

Reply:
There was a missing `padw` for sure. Probably became missing when we changed how `reseed` works. The op stack just before that line is empty so nothing is overwritten but still I would've expect some underflow stack error.